test/jdk/java/awt/print/PrinterJob/PrintTextTest.java

```diff
@@ -240,319 +253,209 @@ private static Font getPhysicalFont() {
+
+        protected final Font textFont;
+        protected final AffineTransform gxTx;
+        protected final String page;
+        protected final boolean useFM;
+        protected final int preferredSize;
+
+        public PrintText(String page, Font font, AffineTransform gxTx, boolean fm, int size) {
+            this.page = page;
+            this.textFont = font;
+            this.gxTx = gxTx;
+            this.useFM = fm;
+            this.preferredSize = size;
+            setBackground(Color.WHITE);
         }
-    }
-
-    public int print(Graphics g, PageFormat pf, int pageIndex) {
-
-        Graphics2D g2d = (Graphics2D)g;
-        g2d.translate(pf.getImageableX(),  pf.getImageableY());
-        g.drawString(page+" "+orient(pf),50,20);
-        g.translate(0, 25);
-        paint(g);
-        return PAGE_EXISTS;
-    }
-
-    public Dimension getMinimumSize() {
-        return getPreferredSize();
-    }
 
-    public Dimension getPreferredSize() {
-        return new Dimension(preferredSize, preferredSize);
-    }
-
-    public void paint(Graphics g) {
-
-        /* fill with white before any transformation is applied */
-        g.setColor(Color.white);
-        g.fillRect(0, 0, getSize().width, getSize().height);
-
-
-        Graphics2D g2d = (Graphics2D) g;
-        if (gxTx != null) {
-            g2d.transform(gxTx);
-        }
-        if (useFM) {
-            g2d.setRenderingHint(RenderingHints.KEY_FRACTIONALMETRICS,
-                                 RenderingHints.VALUE_FRACTIONALMETRICS_ON);
+        private static AttributedCharacterIterator getIterator(String s) {
+            return new AttributedString(s).getIterator();
         }
 
-        g.setFont(textFont);
-        FontMetrics fm = g.getFontMetrics();
-
-        String s;
-        int LS = 30;
-        int ix=10, iy=LS+10;
-        g.setColor(Color.black);
-
-        s = "drawString(String str, int x, int y)";
-        g.drawString(s, ix, iy);
-        if (!textFont.isTransformed()) {
-            g.drawLine(ix, iy+1, ix+fm.stringWidth(s), iy+1);
+        private static String orient(PageFormat pf) {
+            if (pf.getOrientation() == PageFormat.PORTRAIT) {
+                return "Portrait";
+            } else {
+                return "Landscape";
+            }
         }
 
-        iy += LS;
-        s = "drawString(AttributedCharacterIterator iterator, int x, int y)";
-        g.drawString(getIterator(s), ix, iy);
-
-        iy += LS;
-        s = "\tdrawChars(\t\r\nchar[], int off, int len, int x, int y\t)";
-        g.drawChars(s.toCharArray(), 0, s.length(), ix, iy);
-        if (!textFont.isTransformed()) {
-            g.drawLine(ix, iy+1, ix+fm.stringWidth(s), iy+1);
+        @Override
+        public int print(Graphics g, PageFormat pf, int pageIndex) {
+            Graphics2D g2d = (Graphics2D)g;
+            g2d.translate(pf.getImageableX(),  pf.getImageableY());
+            g.drawString(page+" "+orient(pf),50,20);
```

Comment:
```suggestion
            g2d.translate(pf.getImageableX(), pf.getImageableY());
            g.drawString(page + " " + orient(pf), 50, 20);
```

Reply:
Thanks! I've tried to address all of the whitespace quirks, let me know if there is anything else that catches your eye.

Reply:
I was inclined to leave untouched lines as is, there are many formatting inconsistencies in the test code.

If we fix them all, it'll create noise where it'd be hard to see what's really changed.  
I wouldn't mind if the formatting is updated under a separate bug id.

There's no need to revert the updated lines.

Reply:
> This last whitespace feedback came through after the bug was marked as ready for integration, and it looks like the fix then triggered a requirement for re-reviews in the system.

Any change except for clean merging with master requires a re-review.